vertx/src/main/java/com/microsoft/gctoolkit/vertx/VertxChannel.java

```diff
@@ -0,0 +1,25 @@
+package com.microsoft.gctoolkit.vertx;
+
+import com.microsoft.gctoolkit.event.jvm.JVMEvent;
+import com.microsoft.gctoolkit.vertx.io.JVMEventCodec;
+import io.vertx.core.Vertx;
+
+public class VertxChannel {
```

Comment:
needs javadoc + some external docs mentioning this exists.  Does -D command line options allow for override here?